tasks/task04/Collect_tree_leaves.py

```diff
@@ -0,0 +1,36 @@
+def collect_leaves(tree, result=None):
```

Comment:
it would be nice not to pass the result as a parameter, but have it in a function body and when implement the whole function so, that it returns a value that you can append to a result or extend the result with